content/doc/upgrade-guide/2.73.adoc

```diff
@@ -36,12 +36,22 @@ Possible workarounds:
 . Override it for the process running Jenkins, for example using the `-Duser.home=/path/to/directory` parameter to the `java` command, pointing to a writable directory.
 
 
-==== Known Issue: Possible connectivity issue involving ed25519 SSH keys
+==== Known Issue: Randomly occurring connection failures with passphrase-protected ed25519 SSH keys
 
 link:https://issues.jenkins-ci.org/browse/JENKINS-46754[JENKINS-46754]
 
-A possible regression involving SSH authentication using password protected ed25519 keys has been identified.
-More details and the current state of the investigation is available in the linked issue.
+A regression involving authentication using passphrase-protected ed25519 SSH keys has been identified in 2.73.
+It randomly causes failures in features using these keys, such as:
+
+. SSH Slaves Plugin connections
+. Accessing Git repositories with the JGit implementation
+. Accessing Subversion repositories (unconfirmed)
+
+Workarounds for this issue:
+
+. Do not use ed25519 for SSH keys in Jenkins
+. Do not protect ed25519 SSH keys in Jenkins with a passphrase
+. In the case of Git Plugin, switch to the Git CLI implementation
```

Comment:
Since "CLI" might be expanded as "command line interface" or "command line implementation", would it be better to say "Git command line implementation"?